changelog.md

```diff
@@ -1,5 +1,35 @@
 ## Changes from 1.5.x to 1.6.0
 
+Recommended Mesos version is 1.5.0.
```

Comment:
What does this mean?

Reply:
Exactly what it says :) The same thing is in Marathon 1.5 release notes (just with a different Mesos version)